src/TournamentManager.ts

```diff
@@ -861,9 +819,8 @@ export class TournamentManager implements TournamentInterface {
 
 	public async generatePieChart(tournamentId: string): Promise<DiscordAttachmentOut> {
 		const tournament = await this.database.getTournament(tournamentId);
-		const themes = tournament.players.map(p => {
-			// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-			const player = tournament.findPlayer(p)!;
+		// TODO: yikes
```

Comment:
I'm not exactly sure what's yikes here?

Reply:
This was a note I forgot to remove after the n^2 commit, sorry